modules/enabling-etcd-encryption.adoc

```diff
@@ -7,10 +7,10 @@
 
 You can enable etcd encryption to encrypt sensitive resources in your cluster.
 
-// [WARNING]
-// ====
-// Do not back up etcd until the encryption process is complete.
-// ====
+[WARNING]
+====
+It is not recommended to take a backup of etcd until the initial encryption process is complete, otherwise the backup might be only partially encrypted.
```

Comment:
Could you say: _If you backup the etcd database before the initial encryption process is complete, then the backup might be only partially encrypted._ Or similar?

Reply:
@jboxman possibly, but I'd like to work in that it's not recommended to do so. Thoughts?

Reply:
@bergerhoffer, possibly:

> It is not recommended to take a backup of the etcd database until the initial encryption process is complete. If the encryption process is not completed, the backup might be only partially encrypted.

Reply:
+1 thanks @jboxman. I do think that breaking it up into two sentences will be better for translation.